pootle/static/js/auth/components/SocialVerification.js

```diff
@@ -13,6 +13,7 @@ import FormElement from 'components/FormElement';
 import FormMixin from 'mixins/FormMixin';
 
 import { gotoScreen, verifySocial } from '../actions';
+import { t } from 'utils/i18n';
```

Comment:
I would suggest to place this import in the block above, as it is an app-wide import and not a module-relative import.